s2cloudless/cloud_detector.py

```diff
@@ -66,7 +69,7 @@ def __init__(self, threshold=0.4, all_bands=False, average_over=1, dilation_size
             self.dilation_filter = disk(dilation_size)
 
     @property
-    def classifier(self):
+    def classifier(self) -> Any:
```

Comment:
this could be typed even better with a `Protocol`, but i suggest you add that to an issue for future improvements, as it is a bit more difficult

Reply:
issue created